yt_dlp/extractor/vk.py

```diff
@@ -51,11 +51,17 @@ def _perform_login(self, username, password):
                 'Unable to login, incorrect username and/or password', expected=True)
 
     def _download_payload(self, path, video_id, data, fatal=True):
+        url = 'https://vk.com/%s.php' % path
         data['al'] = 1
         code, payload = self._download_json(
-            'https://vk.com/%s.php' % path, video_id,
+            url,
+            video_id,
             data=urlencode_postdata(data), fatal=fatal,
-            headers={'X-Requested-With': 'XMLHttpRequest'})['payload']
+            headers={
+                'Referer': url,
+                'X-Requested-With': 'XMLHttpRequest',
+            },
+        )['payload']
```

Comment:
revert unnecessary changes.
follow our code style

Reply:
No, it is not! You sure you read the guidelines?

Reply:
Thank you. For me it works.